src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -19,6 +19,12 @@ $replicasetdetails-border: #ddd;
 $replicasetdetails-table-message: #000;
 $replicasetdetails-table-cell: #777;
 $replicasetdetails-warning: #ffa500;
+$replicasetdetails-message-font-weight: 400;
```

Comment:
This is the same as $title-font-weight. Can you maybe unify this? Perhaps with changing the name of the variable to cover both cases.